assets/scripts/tracking-onboarding.js

```diff
@@ -1,13 +1,13 @@
-(function(w, d) {
+;(function(w, d) {
   try {
-    const data = d.currentScript.dataset;
-    const tracker = w.Piwik.getTracker(data.matomoUrl, data.matomoSiteId);
-    tracker.enableHeartBeatTimer();
-    tracker.setUserId(w.location.hostname);
-    tracker.setCustomDimension(data.matomoAppId, 'Onboarding');
-    tracker.setCustomUrl('/password');
-    tracker.trackPageView();
-  } catch (err) {
-    console.log('Cannot initialize Matomo:', err);
+    const data = d.currentScript.dataset
+    const tracker = w.Piwik.getTracker(data.matomoUrl, data.matomoSiteId)
+    tracker.enableHeartBeatTimer()
+    tracker.setUserId(w.location.hostname)
+    tracker.setCustomDimension(data.matomoAppId, 'Onboarding')
+    tracker.setCustomUrl('/password')
+    tracker.trackPageView()
+  } catch {
```

Comment:
I think the optional catch binding (`catch {` instead of `catch (e) {`) is not supported by Edge.